commander/src/bootstrapping/templates/lisk-template-ts/templates/app/.gitignore

```diff
@@ -0,0 +1,55 @@
+# General
+~
+.DS_Store
+.project
+__MACOSX/
+*.swp
+*.swo
+ssl/
+tmp/
+
+# Build revision file generated while building a release
+REVISION
+npm-shrinkwrap.json
+
+# Dependency directories
+tsconfig.tsbuildinfo
+node_modules/
+
+# Docs
+docs/jsdoc/
+
+# Logs
+logs/*
+src/logs/
+src/logs.log
+logs.log
+npm-debug.log
+tmux-client-*.log
+framework/test/mocha/network/logs/
+stacktrace*
+
+# IDE directories
+.vscode/
+.idea/
+Session.vim
+
+# Config files
+sftp-config.json
+framework/test/mocha/network/pm2.network.json
+framework/test/mocha/network/configs/
+framework/test/mocha/network/networkTestsLogger.logs
+
+# Coverage directory used by tools like istanbul
+.coverage-unit/
+.coverage/
```

Comment:
i think this we can remove too?

Reply:
We do have test coverage configured in the template. So these are still required.